frontend/database/00212_add_parent_email.sql

```diff
@@ -0,0 +1,3 @@
+-- Emails table
+ALTER TABLE `Emails`
+ADD COLUMN `parent_email` varchar(100) DEFAULT NULL;
```

Comment:
Now that I think well, this field could be a boolean, and it going to be true when U13 users register their parent's email
```suggestion
ADD COLUMN `parent_email` tinyint(1) DEFAULT NULL;
```

Reply:
```suggestion
ADD COLUMN `is_parent_email` tinyint(1) DEFAULT NULL;
```

but why is this needed? isn't it sufficient to know that the user is U13, which means that the email associated with that account is their parent's?

Reply:
@heduenas ^

Reply:
We need to store the parent's email address in a separate field because we need the parent to be able to create their own account so they can monitor their children's activity on omegaup. Otherwise the email value will be repeated in the DB for the children and the parent.

Reply:
if anything, we should instead store the parent email id in the `Users` table as `parent_email_id` instead of creating a new row in Emails with a flag.

Reply:
So then should i close this pR? and create a pr with parent email id in `Users` table as `parent_email_id` ?

Reply:
you can use this same PR to make the change to the `Users` table to avoid closing this and creating a new one.

Reply:
Okayyy!